contentPages/migrations/0035_new_revision_for_draft_landing_pages.py

```diff
@@ -6,9 +6,10 @@
 
 
 def create_revisions(apps, schema_editor):
-    landing_pages = LandingPage.objects.filter(live=False)
-    for landing_page in landing_pages:
-        landing_page.save_revision()
+    pass
+    # landing_pages = LandingPage.objects.filter(live=False)
+    # for landing_page in landing_pages:
+    #     landing_page.save_revision()
```

Comment:
This change shouldn't be committed, we made it locally just to get the migrations to run on a blank db.
However it does want updating so it doesn't break on empty dbs (a ticket for another time).